### DISCUSSION TITLE
Apple Calendar doesn't load

### DISCUSSION THREAD

**Discussion by mentos0007** [MMRIZE/MMM-CalendarExt3Agenda#52] [Q&A]
I connected my public apple-calendar with the basic calendar module but sadly the MMM-CalendarExt3Agenda doesn't show any events. (it works with the public US_holidays). Can someone tell me why?

**Comment by eouia** [comment]
Could you share the ics URL with me? (Without real data, I cannot guess). (eouia0819@gmail.com)

**Comment by Conniverd** [comment]
I had the same problem, the calendar would work in the default calendar module, but would not load in the CalenderEX3Agenda module. The us holidays calendar worked in both. After checking the debug log, I realised that my calendar takes about 25 seconds to load.
I added
`waitFetch: 40000`
to the config, and the calendar loads every time i boot Magicmirror.
